restaurant.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="style.css">
+<title>Page Title</title>
+</head>
+
+<body>
+
+<div class="cursive" class="big;"><h1 id="restName">Airish</h1></div>
```

Comment:
you can use each html attribute only once. Here you use it twice. The right way would be class="cursive big"